iceoryx_hoofs/include/iceoryx_hoofs/log/platform_building_blocks/logger.hpp

```diff
@@ -63,10 +72,23 @@ class Logger : public BaseLogger
 
     ~Logger() = default;
 
+    /// @brief Access to the logger singleton instance
+    /// @return a reference to the active logger
     static Logger& get() noexcept;
 
+    /// @brief Initializes the logger
+    /// @param[in] logLevel is the log level which will be used to determine which messages will be logged. By default
```

Comment:
Remove `is` right after `loglevel` ... reads better.
You have to think about how doxygen generates the help. then you have a website which looks something like

**loglevel:**

is the log level ...

This looks a bit weird